latest_pledgers/latest_pledgers.py

```diff
@@ -4,12 +4,14 @@
 import os
 
 import gspread
-from flask import Flask, jsonify
+from flask import Flask, jsonify, request
+from werkzeug.contrib.cache import SimpleCache
```

Comment:
Nope! [It's a requirement of Flask](https://github.com/mitsuhiko/flask/blob/master/setup.py#L72), so `pip` and other reasonable package managers will install it with Flask. :dancers: